File: Java/EjemploWhileArray/src/EjemploWhileArray.java
import java.util.Scanner;

public class EjemploWhileArray {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Ingrese la cantidad de números del array: ");
        int cantidad = scanner.nextInt();

        int[] array = new int[cantidad];

        System.out.println("Ingrese los elementos del array:");
        int i = 0;
        while (i < cantidad) {
            array[i] = scanner.nextInt();
            i++;
        }

        System.out.println("Elementos del array:");

        int j = 0;
        while (j < cantidad) {
            System.out.println(array[j]);
            j++;
        }

        scanner.close();
    }
}

File: C++/EjemploWhileArray/EjemploWhileArray/EjemploWhileArray/EjemploWhileArray.cpp
#include <iostream>
#include <vector>

int main() {
    std::cout << "Ingrese la cantidad de numeros del arreglo: ";
    int cantidad;
    std::cin >> cantidad;

    std::vector<int> arreglo(cantidad);

    std::cout << "Ingrese los elementos del arreglo:" << std::endl;
    for (int i = 0; i < cantidad; i++) {
        std::cin >> arreglo[i];
    }

    std::cout << "Elementos del arreglo:" << std::endl;
    for (int j = 0; j < cantidad; j++) {
        std::cout << arreglo[j] << std::endl;
    }

    return 0;
}
